gsrs-core-entities/src/main/java/ix/ginas/importers/InputFieldStatistics.java

```diff
@@ -0,0 +1,51 @@
+package ix.ginas.importers;
+
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+public class InputFieldStatistics {
```

Comment:
This is nice, note that it can be used for the SD file stuff too. May have accidentally implemented twice?